geocoding/src/plugin-state.ts

```diff
@@ -0,0 +1,42 @@
+/**
```

Comment:
as written in (https://github.com/jhb-software/payload-plugins/pull/22#issuecomment-3416780825), please remove this and create a custom server component instead, which reads the api key from the payload config.custom and passes it to the client component.